qemu/tests/check_reports_end_offset.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import re
+import json
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    'qemu-img check' should report the end offset of the image.
+    1. Create a test qcow2 image.
+    2. Check there is image end offset and the value with both "humam"
+        and "json" output.
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def _check_result(key, offset, dict):
```

Comment:
Better to use another argument instead of `dict`, since the `dict()` is built-in function